### ISSUE TITLE
suspicious activity block - how to use bypass_security_challenge

### ISSUE THREAD

**Issue by skiwlkr** [InstaPy/InstaPy#5982]
## Expected Behavior
Im having problems with the suspicious activity detection by instagram. Get triggered every 24-36 hours.

**First question:**
Is `Bypass_suspicious_attempt` still a thing? I can't find any information about it. If I implement it in my template it throws an error.
Has `bypass_security_challenge_using='mobile',` the same purpose to bypass the suspicious login attempt? Can't find any information how it should work.
I mean, there is no point bypassing the security challenge, because I have to manually unlock it with a code sent to my mobile. 
I'm sorry if this a stupid question. I don't get the purpose of these functions.

**Second question:**
What can I do to prevent the suspicious activity detection happening in the first place?
I already set up a:
`session.set_quota_supervisor(enabled=True, peak_likes_hourly=40, peak_likes_daily=500, sleep_after=["likes"], sleepyhead=True, stochastic_flow=True, notify_me=True)
`
It works like a charm, but maybe it has to be more aggressive? 
## Current Behavior
 After 24-36 hours of liking, hitting the quota and sleeping suspicious behavior gets detected by instagram and the instapy stops.

Thanks for the help! Appreciate your time!

**Comment by elulcao** [comment]
Your IG has been targeted, you are under suspicious activity block,
so, wait until the cool down period expires; we do not know how long
is it.

The more you use the bot, the more likely you will be banned indefinitely;
there are limits that we cannot be modified or ignored, IG controls that
infrastructure.

**Comment by ghost** [comment]
I think the point of it is to prevent Insatpy from keep trying to send server requests after you are banned.
I haven't got banned so I don't know.

I do have that line in my code however but I've never got to see it work.

The word `'mobile'` should be `'sms'` I believe

**Comment by BoringBears** [comment]
HI @Ardy000  can you share your limit on likes and comments? do you run on a desktop/laptop or mobile? Thanks.

**Comment by skiwlkr** [comment]
> Your IG has been targeted, you are under suspicious activity block,
> so, wait until the cool down period expires; we do not know how long
> is it.
> 
> The more you use the bot, the more likely you will be banned indefinitely;
> there are limits that we cannot be modified or ignored, IG controls that
> infrastructure.

Alright, thank you for your help. I tried now to limit the interaction more aggressive with the Quota supervisor. Instead limiting the likes, I started limiting the server calls, which is preventing the suspiscious activity got triggered so far. 
But still I get this message for every profile instapy try's to open (including my own) probably also from too much activity.

`Failed to get followers count and the page isn't available! the link may be broken, or the page may have been removed`


But, the bot keeps on liking, following, commenting. The grabbed image links are still availabe, which is not perfect but okay.



> The word `'mobile'` should be `'sms'` I believe
 
Thanks! I'll try this out!
